src/Formats/Date.php

```diff
@@ -27,6 +27,9 @@ class Date extends AbstractFormat
      */
     public function validate($data): bool
     {
-        return $this->validateRegex($data, '/^' . self::REGEX . '$/i');
+        if ($this->validateRegex($data, '/^' . self::REGEX . '$/i')) {
```

Comment:
The `explode` is not necessary, just use the `preg_match` function.

```suggestion
        if (!preg_match('/^' . self::REGEX . '$/i', $data, $m)) {
          return false;
        }
        
        return checkdate($m['month'], $m['mday'], $m['fullyear']);
```

Reply:
The suggested change keeps some code from previous commit, resulting in a syntax error.

Reply:
Yes I noticed, my bad. I accidentally clicked commit suggestion. Pushed a fix already!